src/components/ClusterNodes/clusterNodes.service.tsx

```diff
@@ -58,7 +58,7 @@ export const clusterTerminalTypeUpdate = (data): Promise<ResponseType> => {
 }
 
 export const clusterNamespaceList = (): Promise<ResponseType> => {
-    return get(`/${Routes.CLUSTER_NAMESPACE}`)
+    return get(`${Routes.CLUSTER_NAMESPACE}`)
```

Comment:
change this to `Routes.CLUSTER_NAMESPACE` only

Reply:
done